flytekit/types/__init__.py

```diff
@@ -1,3 +1,3 @@
-from flytekit.types.flyte_file import FlyteFile
-from flytekit.types.pandas_schema import PandasSchemaReader, PandasSchemaWriter
+from flytekit.types.file import FlyteFile
```

Comment:
we should clear out this file.

Reply:
we need the pandas_schema imports at least to register the pandas types without necessarily importing the package

Reply:
But if this file isn't empty, if someone does
```
from flytekit.types.file import FlyteFile
```
All the pandas stuff will be loaded as well.

Reply:
ya, for a user to import file, they should need to `from flytekit.types.file import Flytefile`